sympy/functions/combinatorial/factorials.py

```diff
@@ -1057,3 +1058,103 @@ def _eval_is_nonnegative(self):
                 return True
             elif k.is_even is False:
                 return  False
+
+class multinomial(CombinatorialFunction):
+    r"""Implementation of the multinomial coefficient. They can be expressed as
+    a product of binomial coefficients or of factorials:
+
+    .. math:: \frac{(k_1+k_2+\cdots+k_m)!}{k_1!\, k_2! \cdots k_m!} = {k_1\choose k_1}{k_1+k_2\choose k_2}\cdots{k_1+k_2+\cdots+k_m\choose k_m}
+
+    The multinomial coefficients have a direct combinatorial interpretation,
+    as the number of ways of depositing n distinct objects into m distinct bins,
+    with k1 objects in the first bin, k2 objects in the second bin, and so on.
+
+    For the sake of convenience for any negative integer `k` this function
+    will return zero.
+
+    To expand the multinomial when `k` contains a symbol, use either
```

Comment:
This can move to the examples, I think. And there is no need to instruct about `expand` since anyone who is familiar with it will know that polynomials are expanded with `.expand()`.

Reply:
alright

Reply:
(So the idea is to focus docstrings to the function at hand, not provide a broader tutorial on Python/SymPy unless something is likely to be nonintuitive or worthy of bring to remembrance.)